gstreamer-gl/src/enums.rs

```diff
@@ -0,0 +1,175 @@
+// Copyright (C) 2018 Víctor Jáquez <vjaquez@igalia.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use ffi;
+use glib::error::ErrorDomain;
+use glib::translate::*;
+use glib::Quark;
+use GLContextError;
+use GLWindowError;
+
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
+pub enum GLBaseMemoryError {
```

Comment:
Why is this still necessary?

Reply:
because of ``gst_gl_base_memory_error_quark()`` which has to be defined manually because the gir defines ``gst_gl_base_buffer_error_quark()``

Reply:
I fixed that some time last week in GStreamer :)

Reply:
In master, right? I think we should stick to release 1.14 which needs this hack

Reply:
it's just a change in the gir file, making it point to the right quark functions

Reply:
with ``xmlstarlet`` in ``fix.sh``?

Reply:
That would work, yes. I fixed it upstream by generating the correct .gir but until we import that new version, fix.sh would be enough I guess